public/app/features/dashboard/state/DashboardModel.test.ts

```diff
@@ -7,8 +7,16 @@ import { createAdHocVariableAdapter } from '../../variables/adhoc/adapter';
 import { createQueryVariableAdapter } from '../../variables/query/adapter';
 import { createCustomVariableAdapter } from '../../variables/custom/adapter';
 import { expect } from '../../../../test/lib/common';
+import { setTimeSrv, TimeSrv } from '../services/TimeSrv';
 
 jest.mock('app/core/services/context_srv', () => ({}));
+
+const timeSrvMock = ({
+  setAutoRefresh: jest.fn(),
+} as unknown) as TimeSrv;
+
+setTimeSrv(timeSrvMock);
+
```

Comment:
nit: maybe the `setTimeSrv(timeSrvMock);` should be set within the `getTestContext` functions so it's not set in the global scope?

Reply:
indeed, a good idea 🙌🏾 https://github.com/grafana/grafana/pull/34128/commits/02f4f35075001434564b9a2244a10706936f7b8c#diff-0338d437d3a32fb5066ab71687aecdaf0229a778cde30e595af9ff1a39f152d1R823 🚀